docs/visual-testing/sauce-visual.md

```diff
@@ -23,14 +23,53 @@ You can use those standalone or with your Selenium/Appium-based tests. Support f
 
 The best way to integrate Sauce Visual into your existing tests (or write new ones) is to follow the examples listed [in the Visual example repository](https://github.com/saucelabs/visual-examples).
 
-After you have executed your tests, you will find your Visual test results on the Builds History Page:
+After you have executed your tests, you will get the URL to the build in the command line. You can also find your Visual test results on the Builds History Page.
 
 <img src={useBaseUrl('/img/sauce-visual/BuildHistoryPage.png')} alt="Diff History Page"/>
 
+Each Build has an overall status, which is the summary of all the diffs captured within it, along with the build metadata. On the right side, there is a build progress bar, which lets you check at a glance how many diffs need review, are approved, or are rejected.
+
+Selecting one of the builds allows you to get to the Diff Review Page, where you can Approve/Reject detected diffs.
+
+## Reviewing Diffs
+
 Selecting one of the builds allows you to get to the Diff Review Page, where you can Approve/Reject detected diffs.
 
 <img src={useBaseUrl('/img/sauce-visual/DiffReviewPage.png')} alt="Diff Review Page"/>
 
+By default, you will see diffs in a side-by-side view, which lets you easily compare the Current diff against the Baseline. On the right side, you will see a toolbar enabling you to review the diffs. You can:
+
+- Switch from side-by side-to focused view for easier comparison between the Baseline and Current snapshot.
+- Zoom in or out of the snapshots.
+- Zoom to 100% or fit snapshot to the screen.
+- Highlight or hide diffs for easier discovery of the changes.
+
+There are conditional actions you can use, depending on the type of view you have activated:
+
+- If in side-by-side view, you can mirror the zoom and pan actions from the Current snapshot to the Baseline.
+- If in focused view, you can toggle between Baseline and Current diff.
+
+The list below shows the available actions and their keyboard shortcuts.
+
+| Icon                                                                                                                      | Action \[Keyboard Shortcut\]                          |
+| ------------------------------------------------------------------------------------------------------------------------- | ----------------------------------------------------- |
+| <img src={useBaseUrl('/img/sauce-visual/approve.png')} alt="Accept changes icon" width="25"/>                             | Accept changes \[A\] or Undo \[U\]                    |
+| <img src={useBaseUrl('/img/sauce-visual/reject.png')} alt="Reject changes icon" width="25"/>                              | Reject changes \[R\] or Undo \[U\]                    |
+| <img src={useBaseUrl('/img/sauce-visual/cycle-through-diffs.png')} alt="Cycle through snapshots arrows icon" width="25"/> | Cycle through snapshots \[&#x27F5;\] and \[&#x27F6;\] |
```

Comment:
@marstupa I fixed the width for the icons, but the arrows became too small. Can you provide a new one, removing the space between them? Thanks